shared/output/ads/googleads/lib/google/ads/google_ads/v1/services/campaign_service/client.rb

```diff
@@ -176,7 +176,7 @@ def get_campaign request, options = nil
               # @overload mutate_campaigns(customer_id: nil, operations: nil, partial_failure: nil, validate_only: nil)
               #   @param customer_id [String]
               #     The ID of the customer whose campaigns are being modified.
-              #   @param operations [Google::Ads::GoogleAds::V1::Services::CampaignOperation | Hash]
+              #   @param operations [Array<Google::Ads::GoogleAds::V1::Services::CampaignOperation | Hash>]
```

Comment:
`>` for `Array` should follow `Operation`, not `Hash`.

Reply:
Oh, maybe I'm wrong about that, but still not sure this works with YARD as it is now.

Reply:
Actually, it should follow `Hash` because it's an array of either operation objects or hashes. So that part is correct as is.

But now that you drill into this, I realize there are still things wrong. The yard syntax is to use commas rather than vertical pipes. Also, none of the params under the overload clauses are actually getting rendered in yard to begin with (probably for unrelated reasons that I haven't sussed out yet).

This PR is fast going to turn into a fix-everything-wrong-with-yard PR which I want to avoid. Can we accept this PR which fixes some things but not others, and most importantly unblocks generation of the recommender service? I'll file additional issues for the other bugs we're seeing.

Reply:
Agreed